tests/managers/account_tests.py

```diff
@@ -21,15 +21,15 @@ def test_get_summary(self):
         self.assert_called_with('SoftLayer_Account', 'getObject')
 
     def test_get_planned_upcoming_events(self):
-        self.manager.get_upcoming_events("PLANNED")
+        self.manager.get_upcoming_events("PLANNED", "")
```

Comment:
Once you update the `add_event_filter` function to have a default `min_date` revert these tests please.

Reply:
Done